src/App.js

```diff
@@ -11,7 +13,11 @@ function App() {
         name="NC Games"
         paragraph="No boredom on our board games."
       />
-      <ReviewList />
+      <Routes>
```

Comment:
you've routed your reviews components to the right url but there's no link to these urls anywhere on your app, can you add them in please